tests/integration/suite/actors/reminders/rebalancing.go

```diff
@@ -406,19 +383,16 @@ func (i *rebalancing) reportStatusToPlacement(ctx context.Context, stream placem
 
 	errCh := make(chan error)
 	go func() {
-		var sent bool
```

Comment:
We can't have a go routine leak here because we are waiting [below](https://github.com/dapr/dapr/pull/7296/files#diff-dcc3bd43d7bae701ab1f598512043ae509049e0d20132ea88eb3a414d8b9c3a5L429) for this go routine to return. A context cancelled will result in everything (including the stream here) closing. I've increased `errCh` to a size of 1 so it can always write to the channel.

Reply:
I added it because there was a leak when I wrote it :) When you cancel the context, `strem.Recv()` returns an error indicating the connection has been closed.

If you made `errCh` buffered then that works